src/yetibot/commands/aws.clj

```diff
@@ -3,167 +3,169 @@
     [taoensso.timbre :refer [info]]
     [yetibot.core.hooks :refer [cmd-hook]]
     [yetibot.api.aws :as aws]
-    [yetibot.commands.aws.formatters :refer [format-response]]))
+    [yetibot.commands.aws.formatters :refer [format-iam-response format-s3-response]]
+    [lambdaisland.uri :refer [uri]]
+    [clojure.string :as str]))
 
 (defn iam-create-group-in-path-cmd
   "aws iam create-group <path> <group-name> # Creates an aws IAM group named <group-name> within the specified <path>"
   {:yb/cat #{:util :info}}
   [{[_ path group-name] :match}]
   (-> (aws/iam-create-group path group-name)
       (with-meta {:aws/type :aws.type/CreatedGroup})
-      format-response))
+      format-iam-response))
 
 (defn iam-create-group-cmd
   "aws iam create-group <group-name> # Creates an aws IAM group named <group-name> within the default path /"
   {:yb/cat #{:util :info}}
   [{[_ group-name] :match}]
   (-> (aws/iam-create-group "/" group-name)
       (with-meta {:aws/type :aws.type/CreatedGroup})
-      format-response))
+      format-iam-response))
 
 (defn iam-create-user-cmd
   "aws iam create-user <user-name> # Creates an aws IAM user named <user-name>"
   {:yb/cat #{:util :info}}
   [{[_ user-name] :match}]
     (-> (aws/iam-create-user "/" user-name)
         (with-meta {:aws/type :aws.type/CreatedUser})
-      format-response))
+        format-iam-response))
 
 (defn iam-create-user-in-path-cmd
   "aws iam create-user <path> <user-name> # Creates an aws IAM user named <user-name> within the specified <path> prefix"
   {:yb/cat #{:util :info}}
   [{[_ path user-name] :match}]
   (-> (aws/iam-create-user path user-name)
       (with-meta {:aws/type :aws.type/CreatedUser})
-      format-response))
+      format-iam-response))
 
 (defn iam-add-user-to-group-cmd
   "aws iam add-user-to-group <group-name> <user-name> # Adds an aws IAM user <user-name> to an IAM group <group-name>"
   {:yb/cat #{:util :info}}
   [{[_ group-name user-name] :match}]
   (-> (aws/iam-add-user-to-group group-name user-name)
       (with-meta {:aws/type :aws.type/UserAddedToGroup})
-      format-response))
+      format-iam-response))
 
 (defn iam-remove-user-from-group-cmd
   "aws iam remove-user-from-group <group-name> <user-name> # Removes an aws IAM user <user-name> from the IAM group <group-name>"
```

Comment:
Looks like args are swapped here too, same as `add-user-to-group`.

![image](https://user-images.githubusercontent.com/86107/79150866-b856b000-7d86-11ea-9b86-4946fafa9794.png)

Reply:
I will review those, I may have had a bad `copy/paste` somewhere

Reply:
This one is also _fixed_